run_macos.sh

```diff
@@ -27,9 +36,8 @@ if [ $? -ne 0 ]; then
 fi
 echo "Dependencies installed successfully."
 
-# Step 4: Run the application
-echo "Starting the application..."
-python server.py
+echo "Starting the application with ConfigDirectory=$CONFIG_DIR and User=$USER..."
+python3 server.py "$CONFIG_DIR" "$USER"
```

Comment:
On Linux, you should be able to leave all the argument parsing out of the runner script and just call `python3 server.py "$@"`, letting server.py do all the parsing. 

I've been using run_macos.sh on Fedora with zsh and it's work flawlessly. Hopefully the above suggestions works on MacOS, but I have no clue how it would be done on Windows.

Reply:
I was thinking of separating out a mac and linux sh for stuff like that in the past, but now I'm torn. On one hand, that would be cleaner for Linux to use your syntax (assuming it doesn't work for Mac... it might), but on the other hand the fact that you're able to use run_macos.sh makes me want to just rename that file "run_wilmer.sh" and have windows be "run_wilmer.bat".

Either way, let me know how the SH does in its current state. I've had more chance to test this on Windows than on Mac so I could have broken something in .sh file.

Reply:
I tried your suggestion on Mac and it worked. I've updated the bash script with it. If it works for you, I'm just gonna rename that file "run_wilmer.sh" for both mac and linux